pdpserver/control.go

```diff
@@ -164,3 +164,10 @@ func (s *server) Apply(ctx context.Context, in *pb.Update) (*pb.Response, error)
 
 	return res, err
 }
+
+func (s *server) NotifyReady(ctx context.Context, m *pb.Empty) (*pb.Response, error) {
+	log.Info("Got notified about readiness")
+	s.start<-true
```

Comment:
Instead of stopping there and then resuming here it's better to make serveRequests function and just call it here